### PR TITLE
fix #60472 duplichecker.com

### LINKED ISSUE #60472: duplichecker.com
### Issue URL (Ads)
 [https://www.duplichecker.com/cloaking-checker.php](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fwww.duplichecker.com%2Fcloaking-checker.php)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard Browser extension v3.4.31
Browser: | Chrome
Filters: | <b>Ad Blocking:</b><br/> AdGuard Base<br/><br/><b>Privacy:</b><br/> AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/> AdGuard Social Media<br/><br/><b>Language-specific:</b><br/> AdGuard German

### PR DESCRIPTION
**Pull request by stanislav-atr**
#60472

### CODE REVIEW

File: EnglishFilter/sections/css_extended.txt [hunk 1]
```diff
@@ -540,7 +543,7 @@ oddsdigest.com#?#.td-ss-main-sidebar > .td_block_template_1:has(> h4 > span:cont
 reblop.com#?#.td-block-row > div[class^="td-block-span"]:has(a[href^="https://t.frtyo.com/"])
 ! https://github.com/AdguardTeam/AdguardFilters/issues/33446
 duplichecker.com#$?#.adsbygoogle { remove: true; }
-duplichecker.com#?#.side-bar > .search-panel-sec:has(> .adds_box)
+.com#?#.side-bar > .search-panel-sec:has(> .adds_box)
```
Comment (AdamWr) [inline EnglishFilter/sections/css_extended.txt:546]: It seems that there is something wrong with this rule :)

By the way, they display different ads if they detect ad blocker, so I think that we could also change this rule:
https://github.com/AdguardTeam/AdguardFilters/blob/9c93e652efbf054fc788271ed86cd8ce951595f6/EnglishFilter/sections/antiadblock.txt#L1071

To for example something like:
`duplichecker.com#%#//scriptlet("set-constant", "google_jobrunner", "falseFunc")`
or/and also add this:
`duplichecker.com#%#//scriptlet("prevent-setInterval", "isAdBlockActive")`

Comment (zloyden) [reply EnglishFilter/sections/css_extended.txt:546]: @AdamWr for `different ads if they detect ad blocker` this is ok `duplichecker.com#%#//scriptlet("prevent-setInterval", "isAdBlockActive")`
These rules should be enough for content blockers
```
duplichecker.com#@#.adsbygoogle
@@||pagead2.googlesyndication.com/pagead/js/adsbygoogle.js$domain=duplichecker.com
@@||pagead2.googlesyndication.com/pagead/js/*/show_ads_impl*.js$domain=duplichecker.com
```

File: EnglishFilter/sections/css_extended.txt [hunk 2]
```diff
@@ -27,6 +27,10 @@ wormate.io#?#.line-center > .column-right:has(> div[id] > .label:contains(/^Adve
 wormate.io#?##game-cont > #results-view > div[id][style]:has(> .label:contains(/^Advertisement:$/))
 ! https://github.com/AdguardTeam/AdguardFilters/issues/59371
 pngegg.com#$?#.list_ads { remove: true; }
+! https://github.com/AdguardTeam/AdguardFilters/issues/60472
+duplichecker.com#?#div > div:has(> span > a[href="https://www.duplichecker.com/linksmgm1"])
+duplichecker.com#?#div > div:has(> div > div#ldo6uv2f > div > span > a[href="https://www.duplichecker.com/linksmgm1"])
+duplichecker.com#?#body > div.search_bar_area:has(> div > div > div > span > a[href="https://www.duplichecker.com/linksmgm1"])
```
Comment (zloyden) [inline EnglishFilter/sections/css_extended.txt:33]: You should re-write it to something like these for leftovers:
```
duplichecker.com##div[class*="aadd_boox"]
duplichecker.com#?#body > div[class]:has(> div.container > div.clearfix + div[class] > div[class*="aadd_boox"])
duplichecker.com#?#.side-bar > div:not([class]):has(> div span.text-center > a[target="_blank"][rel="nofollow"])
```
`div > div` we don't use it in our filters, look at examples on `css_extended` rules. It should be more specific.
2nd rule - 2nd part. This `div#ldo6uv2f` seems to have a random id. You should avoid it in future.
3rd rule - 2nd part. Too much `div`, some of them you can skip.

**Review by zloyden — CHANGES_REQUESTED** [review]

File: EnglishFilter/sections/css_extended.txt (hunk 2 above)
Comment (stanislav-atr) [reply EnglishFilter/sections/css_extended.txt:33]: ok, omw
#ldo6uv2f seemed pretty static for me, i could change it to something like `div[id]`

**Comment by stanislav-atr** [comment]
I could make some rules shorter using `whitespace` selector instead of `>` few times.

**Review by zloyden — APPROVED** [review]

**Review by BlazDT — APPROVED** [review]

**Comment by zloyden** [comment]
> I could make some rules shorter using `whitespace` selector instead of `>` few times.

Yeah, you can do it to avoid some unnecesary elements in future, but you have to check if it applies correctly to the code.

**Review by AdamWr — APPROVED** [review]

**Pull request merged**